gravitee-apim-console-webui/src/components/notifications/notifications-list/notification-details/notifications-details-reusable.component.html

```diff
@@ -0,0 +1,28 @@
+<div>
+  <div class="md-headline notifications-form__title">
```

Comment:
md-headline is usefull ?

Reply:
It sets correct title size, I can share screenshot if you want